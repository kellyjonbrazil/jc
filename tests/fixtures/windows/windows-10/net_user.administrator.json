{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator",
      "comment": "Built-in account for administering the computer/domain",
      "country_region_code": "000 (System Default)",
      "account_active": false,
      "account_expires": "Never",
      "password_last_set": "2022-04-04T20:06:30",
      "password_expires": "Never",
      "password_changeable": "2022-04-04T20:06:30",
      "password_required": false,
      "user_may_change_password": true,
      "workstations_allowed": "All",
      "last_logon": "Never",
      "logon_hours_allowed": "All",
      "local_group_memberships": [
        "Administrators"
      ],
      "global_group_memberships": []
    }
  ]
}
