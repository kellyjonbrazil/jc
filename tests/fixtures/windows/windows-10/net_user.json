{
  "domain": "",
  "user_accounts": [
    {
      "user_name": "Administrator"
    },
    {
      "user_name": "DefaultAccount"
    },
    {
      "user_name": "Guest"
    },
    {
      "user_name": "user1"
    },
    {
      "user_name": "sshd"
    },
    {
      "user_name": "WDAGUtilityAccount"
    }
  ],
  "account_origin": "\\\\DESKTOP-WIN10-PRO"
}
