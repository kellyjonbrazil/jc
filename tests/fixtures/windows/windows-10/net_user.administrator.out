User name                    Administrator
Full Name
Comment                      Built-in account for administering the computer/domain
User's comment
Country/region code          000 (System Default)
Account active               No
Account expires              Never

Password last set            4/4/2022 8:06:30 PM
Password expires             Never
Password changeable          4/4/2022 8:06:30 PM
Password required            No
User may change password     Yes

Workstations allowed         All
Logon script
User profile
Home directory
Last logon                   Never

Logon hours allowed          All

Local Group Memberships      *Administrators
Global Group memberships     *None
The command completed successfully.
